app/models/message.rb

```diff
@@ -21,16 +21,25 @@ class Message < ApplicationRecord
 
   validates :text, presence: true
 
-  def notify
+  def after_creation
```

Comment:
I would use a callback `after_create :notify, :increment_unread` and remove this method.

Reply:
I tried it like this at first, but was not viable since it requires to have the user set first, which is done after creation, by the controller... unless there is a way in which I can set both conversation and user ids at the same time, but I couldnt find any,  going through  user.conversations.messages.create! doesnt work. 
@leandrodzp cant tag 😢